node-graph/gcore/src/vector/manipulator_group.rs

```diff
@@ -146,6 +146,7 @@ impl ManipulatorGroup {
 
 		// If the anchor isn't selected, but both handles are, drag only handles
 		if self.both_handles_selected() {
+			self.editor_state.mirror_angle_between_handles = false;
```

Comment:
This line is to cancel handle angle mirroring when both handles are dragged without the anchor - is this line enough or do I have to make other changes?

Reply:
Does the behavior of the tool reflect the change correctly? If so it is likely all that needs to be done. Test selecting both handles & the anchor and dragging as well. That is another situation to consider.